source/systems/gaea_user_guide.rst

```diff
@@ -2141,90 +2403,60 @@ list below.
 
 Filesystems that GCP supports locally from Gaea:
 
-- eslogin
+* eslogin
 
-.. note::
+ - /lustre/f2
```

Comment:
F2 is gone.  This should be `/gpfs/f5`

Reply:
That and /gpfs/f6.  

I'm sure we need C6 information on the page as well.